types/react-amplitude/react-amplitude-tests.ts

```diff
@@ -0,0 +1 @@
+import Amplitude from 'react-amplitude';
```

Comment:
You can use this code as in test file: https://github.com/rorygarand/react-amplitude#usage